### DISCUSSION TITLE
Discussing PartDesign tickets (closing duplicates or overlapping)

### DISCUSSION THREAD

**Discussion by luzpaz** [FreeCAD/FreeCAD#11433] [Discussions that will turn in to issues but need refining]
### Cloned: 
https://github.com/FreeCAD/FreeCAD/issues/5720 [Problem] PartDesign: Duplicate body creates a mess
https://github.com/FreeCAD/FreeCAD/issues/6741 [Bug] PD primitives cannot be copied to another body 
https://github.com/FreeCAD/FreeCAD/issues/6985 [Bug] Copy & Paste of PartDesign feature pastes feature outside the Body object
https://github.com/FreeCAD/FreeCAD/issues/9154 [Problem] PartDesign: When cutting a cloned body from a cloned source body the "parent" of the cutting clone is added and cut twice
https://github.com/FreeCAD/FreeCAD/issues/9251 [Issue] Copy Paste of simple element is dauntingly complex.
https://github.com/FreeCAD/FreeCAD/issues/11011 [Problem] Clone of a clone does not inherit the colors 

### Mirrored
https://github.com/FreeCAD/FreeCAD/issues/5678 Add Mirror body feature to PartDesign
https://github.com/FreeCAD/FreeCAD/issues/5705 [Feature Request] PartDesign: Add a Mirror body tool 
https://github.com/FreeCAD/FreeCAD/issues/8439 [Problem] PartDesign: Mirror doesn't fail when Originals is emptied 
https://github.com/FreeCAD/FreeCAD/issues/10468 [Problem] Deleting feature allocates dependent feature to the wrong location
https://github.com/FreeCAD/FreeCAD/issues/10528 [Problem] Access violation with PD Mirror
https://github.com/FreeCAD/FreeCAD/issues/10533 [Problem] PartDesign: Mirrored feature mirror plane selection doesn't work

**Comment by M4xTheMan** [comment]
Could you explain how help could look like?
